test/powershell/Modules/Microsoft.PowerShell.Management/SymLinkTests.Tests.ps1

```diff
@@ -0,0 +1,68 @@
+function Clean-State {
+    
+    if (Test-Path $FullyQualifiedLink)
+    {
+	Remove-Item $FullyQualifiedLink -Force
+    }
+    
+    if (Test-Path $FullyQualifiedChildFolder)
+    {
+        Remove-Item $FullyQualifiedChildFolder -Force
+    }
+
+    if (Test-Path $FullyQualifiedFolder)
+    {
+        Remove-Item $FullyQualifiedFolder -Force
+    }
+
+    if (Test-Path $tmpDirectory)
+    {
+        Remove-Item $tmpDirectory -Force
+    }
+}
+
+Describe "New-Item" -Tags "CI" {
```

Comment:
>"CI" [](start = 26, length = 4)

this also needs the tag "RequireAdminOnWindows" since you can't create symlinks on windows unless you're elevated